<launch>
  <arg name="input_twist_with_covariance_topic" default="sensing_twist_with_covariance" description="input twist with covariance topic name"/>
  <arg name="output_twist_with_covariance_topic" default="localization_twist_with_covariance" description="output twist with covariance topic name"/>
  <arg name="message_timeout_sec" default="0.2" description="delay tolerance time for message"/>

  <node pkg="twist_estimator" exec="twist_estimator" name="twist_estimator" output="screen">
    <remap from="in_twist_with_covariance" to="$(var input_twist_with_covariance_topic)"/>
    <remap from="out_twist_with_covariance" to="$(var output_twist_with_covariance_topic)"/>
    <param name="message_timeout_sec" value="$(var message_timeout_sec)"/>
  </node>
</launch>
